tests/functional_composition/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -512,15 +476,12 @@ const resolve = createSelector(
 =====================================output=====================================
 import { createSelector } from "reselect";
 
-const resolve = createSelector(
-  getIds,
-  getObjects,
-  (ids, objects) => ids.map(id => objects[id])
+const resolve = createSelector(getIds, getObjects, (ids, objects) =>
+  ids.map(id => objects[id])
 );
 
-const resolve = createSelector(
-  [getIds, getObjects],
-  (ids, objects) => ids.map(id => objects[id])
+const resolve = createSelector([getIds, getObjects], (ids, objects) =>
+  ids.map(id => objects[id])
```

Comment:
Previously looks better

Reply:
🤷‍♂️ can’t win every battle. Lemme just mention that all the ramda examples are now formatted [exactly as they are in the documentation](https://ramdajs.com/docs/#compose) 🙂🙃🙂🙃